tests/wicked/before_test.pm

```diff
@@ -46,11 +46,13 @@ sub run {
         zypper_call('--quiet in dhcp-server', timeout => 200);
         if (check_var('WICKED', '2nics')) {
             record_info('Info', 'Set up 2nics DHCP and second ip');
-            assert_script_run(sprintf("ip a a %s dev %s", $self->get_ip(type => 'second_card'), $ctx->iface()));
             $self->get_from_data('wicked/dhcp/dhcpd_2nics.conf', '/etc/dhcpd.conf');
```

Comment:
as far as I remember this was added to before test exactly for second card scenario , so we can remove it with this PR

Reply:
ok

Reply:
done